prime-router/src/main/kotlin/cli/main.kt

```diff
@@ -130,7 +130,7 @@ class GenerateDocs : CliktCommand(
         metavar = "<path>",
         help = "write documentation to this file (should not include extension)"
     )
-    private val defaultOutputDir = "docs/schema_documentation"
+    private val defaultOutputDir = "generated/schema_documentation"
```

Comment:
I think `docs/generated` might make more sense? I'm stretching here to find anything that needs feedback :)

Reply:
100% don't feel strongly and would actually defer to @JessicaWNava or at @arnejduranovic.  My initial thought was that this directory is now intended to just contain the curated documentation we write to maintain reporstream and that those schema_documentation are aligned with that.

Reply:
And just to provide more context, I believe all that schema documentation is **not** for the universal pipeline so it out of place in docs/ anyway

Reply:
Do you believe it belongs somewhere else then? If we are going to keep it here, I still think a dir change would be nice.

Reply:
I don't feel strongly outside of the fact that they should live somewhere else since they're only for legacy pipeline.  Open to suggestions for the directory name though,  I have no clue who uses these so `generated` seems as good as any to me.

Reply:
Hmm ya I don't really want to put this in docs since its covid pipeline... how about we put it in deprecated? I'll ask at Eng sync...

Reply:
I agree with moving this to deprecated. Once we migrate everybody to the UP we won't need this anymore.